docs/advanced-swapping/smart-swap-options.mdx

```diff
@@ -5,56 +5,45 @@ description: 'This page introduces the Smart Swap functionality provided by the
 
 ## Introduction
 
-Smart Swap refers to a broad range of functionality we're building out to improve swap speed, price, and customization.
+Smart Swap refers to a feature set that improves swap speed, price, and control.
 
-At this time, the main use case for smart swap is improving trade execution price by enabling third-party routers (e.g. Hallswap) and trade splitting. But much more is on the way!
-
-This doc covers:
-
-- All the different features Smart Swap enables
-- Describes how to use each of them
+It currently allows for:
+- [External routers](#feature-use-external-routers-to-improve-price-execution) (e.g. Hallswap and Osmosis SQS) for better price execution
+- [Route splitting](#feature-route-splitting) for better price execution
+- [EVM swaps](#feature-evm-swaps)
 
  <Warning>
- **`@skip-router` 4.0.0+ required**
+If you're using the deprecated `@skip-router` library, you must use version v4.0.0+ to enable Smart Swap.
 
-If you're using the deprecated `@skip-router` library, you must use v4.0.0 or later to use Smart Swap.
-
-We advise all integators to use the new `@skip-go/client` [TypeScript Package](https://www.npmjs.com/package/@skip-go/client), which is being actively maintained and developed.
+We strongly reccomend using the `@skip-go/client` [TypeScript package](https://www.npmjs.com/package/@skip-go/client), which is actively maintained.
 
 </Warning>
 
-## How to Use Smart Swap
-
-All Smart Swap features are controlled with a single configuration object:
 
-1. `@skip-go/client` Package: `smartSwapOptions` object passed in`SkipClient.route` options `SkipClient.msgsDirect` options
-2. REST API: `smart_swap_options` parameter passed in `/v2/fungible/route` and `/v2/fungible/msgs_direct`
+The rest of this document will show you how to use Smart Swap with the `@skip-go/client` library. The only changes you'll notice between this context and the REST API are naming conventions.
 
-Read on for more info about the individual features and how to use each of them.
+# Smart Swap Features
 
-(For the rest of this doc, examples will show how to use Smart Swap with the `@skip-go/client` library by configuring it on `SkipClient.route`. The only changes you'll notice between this context and the REST API is naming conventions)
+Set your Smart Swap settings in your `skipClient` function call our REST API request body.
```

Comment:
I think there's some typo here? the sentence doesn't make sense to me

Reply:
good catch, should just be removed.